distributed/utils.py

```diff
@@ -44,7 +44,7 @@
 
 
 try:
-    from dask import thread_state
+    from dask.context import thread_state
```

Comment:
There has been one distributed release, and no dask release since this was added. The users this backwards incompatible change affects are those running the new distributed release with dask master. If we wanted to be 100% backwards compatible we could check for `thread_state` in `dask.base` if it's not found in `dask.context`. Didn't seem worth it to me though.

Reply:
Yeah, I wouldn't worry about it.  The next release will probably pin each to the other.